tests/helpers/kubernetes/agent.py

```diff
@@ -105,9 +105,8 @@ def deploy(self, agent_yaml=None):
                 print(f"Created agent configmap:\n{configmap_base}")
 
                 daemonset_base = load_resource_yaml(AGENT_DAEMONSET_PATH)
-                daemonset_base["spec"]["template"]["spec"]["containers"][0]["imagePullPolicy"] = "Always"
                 daemonset_base["spec"]["template"]["spec"]["containers"][0]["image"] = self.agent_image_name
-                daemonset_base["spec"]["template"]["spec"]["containers"][0]["imagePullPolicy"] = "Always"
+                daemonset_base["spec"]["template"]["spec"]["containers"][0]["imagePullPolicy"] = "IfNotPresent"
```

Comment:
Why??

Reply:
Because when the image that gets created with `make image` (quay.io/signalfx/signalfx-agent-dev) gets pushed to the docker instance having it set to Always will try to pull it from quay.io. Thinking about it more I guess this could break for the case where it's pulling from localhost:5000 repository. Might have to make the value configurable through pytest option? @jchengsfx

Reply:
How are you running the integration tests?

Reply:
`pytest  --no-use-minikube --kube-context minikube --agent-image-name quay.io/signalfx/signalfx-agent-dev:4.7.8-post`